stdlib/CMakeLists.txt

```diff
@@ -164,6 +164,10 @@ option(SWIFT_ENABLE_REFLECTION
   "Build stdlib with support for runtime reflection and mirrors."
   TRUE)
 
+option(SWIFT_BUILD_PRIVATE_STDLIB
```

Comment:
I slightly think that `SWIFT_STDLIB_BUILD_PRIVATE` might be a better spelling for the option, but 🤷‍♂️ .

Reply:
Good point -- tackled in d39805b3ff603657923f344b92f8fe676915a522